src/selectionHandler.ts

```diff
@@ -50,13 +50,18 @@ function eventDataOf(completions: any, currentCompletion: any, editor: TextEdito
     });
 
     const length = currentCompletion.length;
-    const netLength = editor.selection.anchor.character - position.character;
+    const netLength = editor.document.getText(new vscode.Range(position, editor.selection.anchor))
```

Comment:
tested it? worked?

Reply:
i console logged the text at the document in the calculated position (from the beginning of the line) and it looked quite right... although i dont fully understand what is the expected behaviour in the first place